install_scripts/templates/docker-install/19-03-ce.sh

```diff
@@ -0,0 +1,499 @@
+#!/bin/sh
+set -e
+
+{% if docker_version -%}
+# this will hardcode the docker version in the script courtesy of replicated
+VERSION="{{ docker_version }}"
+{%- endif %}
+
+# Docker CE for Linux installation script
+#
+# See https://docs.docker.com/install/ for the installation steps.
+#
+# This script is meant for quick & easy install via:
+#   $ curl -fsSL https://get.docker.com -o get-docker.sh
+#   $ sh get-docker.sh
+#
+# For test builds (ie. release candidates):
+#   $ curl -fsSL https://test.docker.com -o test-docker.sh
+#   $ sh test-docker.sh
+#
+# NOTE: Make sure to verify the contents of the script
+#       you downloaded matches the contents of install.sh
+#       located at https://github.com/docker/docker-install
+#       before executing.
+#
+# Git commit from https://github.com/docker/docker-install when
+# the script was uploaded (Should only be modified by upload job):
+SCRIPT_COMMIT_SHA="UNKNOWN"
```

Comment:
can you add a sha here from the docker repo? https://github.com/docker/docker-install/commit/b50fac9805c3d95fb5e8b16d57b19d333e972e20

Reply:
changed